cypress/component/rich-text/RichTextEditor.HR.spec.ts

```diff
@@ -13,9 +13,12 @@ describe('Rich Text Editor - HR', { viewportHeight: 2000, viewportWidth: 1000 },
   const expectDocumentToBeEmpty = () => richText.expectValue(undefined);
 
   function addBlockquote(content = '') {
-    richText.editor.click().type(content);
+    richText.editor.click();
+    richText.editor.should('be.focused');
```

Comment:
What do you think about adding a helper function that does these three steps and you just pass in the text to be typed?

Reply:
🤔 fair point, although I think being focused is a natural assumption if you're typing. Maybe something like "focusAndType"?

Reply:
We may also want to do something like this for markdown and JSON editors, not sure if those have been flakey though

Reply:
Yep, actually, used almost the same name before reading your comment 😀 Also, added few (potentially) redundant checks, but should help to make it more stable.

Reply:
I feel like with cypress you can never have too much redundancy :D